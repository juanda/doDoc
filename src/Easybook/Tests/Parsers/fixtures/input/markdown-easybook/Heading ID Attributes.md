# Header 1

## Header 2

### Header 3

#### Header 4

##### Header 5

###### Header 6

Repeat the same headings to ensure that the ID are unique:

# Header 1

## Header 2

### Header 3

#### Header 4

##### Header 5

###### Header 6

Repeat again the same headings, but with the complete atx-style syntax:

# Header 1 #

## Header 2 ##

### Header 3 ###

#### Header 4 ####

##### Header 5 #####

###### Header 6 ######

Repeat the same headings but combine setext and atx styles:

Header 1
========

Header 2
--------

### Header 3

#### Header 4

##### Header 5

###### Header 6

Use custom ID attributes:

# Header 1 {#my-custom-id-1}

## Header 2 {#my-custom-id-2}

### Header 3 {#my-custom-id-3}

#### Header 4 {#my-custom-id-4}

##### Header 5 {#my-custom-id-5}

###### Header 6 {#my-custom-id-6}

Use custom ID attributes with the complete atx-style headers:

# Header 1 # {#my-other-custom-id-1}

## Header 2 ## {#my-other-custom-id-2}

### Header 3 ### {#my-other-custom-id-3}

#### Header 4 #### {#my-other-custom-id-4}

##### Header 5 ##### {#my-other-custom-id-5}

###### Header 6 ###### {#my-other-custom-id-6}

Use custom ID attributes combining setext and atx styles:

Header 1 {#my-another-custom-id-1}
========

Header 2 {#my-another-custom-id-2}
--------

### Header 3 ### {#my-another-custom-id-3}

#### Header 4 #### {#my-another-custom-id-4}

##### Header 5 ##### {#my-another-custom-id-5}

###### Header 6 ###### {#my-another-custom-id-6}

