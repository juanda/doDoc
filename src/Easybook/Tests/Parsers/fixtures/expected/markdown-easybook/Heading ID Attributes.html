<h1 id="header-1-3">Header 1</h1>

<h2 id="header-2-3">Header 2</h2>

<h3 id="header-3">Header 3</h3>

<h4 id="header-4">Header 4</h4>

<h5 id="header-5">Header 5</h5>

<h6 id="header-6">Header 6</h6>

<p>Repeat the same headings to ensure that the ID are unique:</p>

<h1 id="header-1-4">Header 1</h1>

<h2 id="header-2-4">Header 2</h2>

<h3 id="header-3-3">Header 3</h3>

<h4 id="header-4-3">Header 4</h4>

<h5 id="header-5-3">Header 5</h5>

<h6 id="header-6-3">Header 6</h6>

<p>Repeat again the same headings, but with the complete atx-style syntax:</p>

<h1 id="header-1-5">Header 1</h1>

<h2 id="header-2-5">Header 2</h2>

<h3 id="header-3-4">Header 3</h3>

<h4 id="header-4-4">Header 4</h4>

<h5 id="header-5-4">Header 5</h5>

<h6 id="header-6-4">Header 6</h6>

<p>Repeat the same headings but combine setext and atx styles:</p>

<h1 id="header-1">Header 1</h1>

<h2 id="header-2">Header 2</h2>

<h3 id="header-3-5">Header 3</h3>

<h4 id="header-4-5">Header 4</h4>

<h5 id="header-5-5">Header 5</h5>

<h6 id="header-6-5">Header 6</h6>

<p>Use custom ID attributes:</p>

<h1 id="my-custom-id-1">Header 1</h1>

<h2 id="my-custom-id-2">Header 2</h2>

<h3 id="my-custom-id-3">Header 3</h3>

<h4 id="my-custom-id-4">Header 4</h4>

<h5 id="my-custom-id-5">Header 5</h5>

<h6 id="my-custom-id-6">Header 6</h6>

<p>Use custom ID attributes with the complete atx-style headers:</p>

<h1 id="my-other-custom-id-1">Header 1</h1>

<h2 id="my-other-custom-id-2">Header 2</h2>

<h3 id="my-other-custom-id-3">Header 3</h3>

<h4 id="my-other-custom-id-4">Header 4</h4>

<h5 id="my-other-custom-id-5">Header 5</h5>

<h6 id="my-other-custom-id-6">Header 6</h6>

<p>Use custom ID attributes combining setext and atx styles:</p>

<h1 id="my-another-custom-id-1">Header 1</h1>

<h2 id="my-another-custom-id-2">Header 2</h2>

<h3 id="my-another-custom-id-3">Header 3</h3>

<h4 id="my-another-custom-id-4">Header 4</h4>

<h5 id="my-another-custom-id-5">Header 5</h5>

<h6 id="my-another-custom-id-6">Header 6</h6>
